frontend/src/app/pages/signin/signin.page.component.html

```diff
@@ -1,2 +1,31 @@
-<h1>Sign In Page</h1>
-<button (click)="signin()">Sign In</button>
\ No newline at end of file
+<!-- Add the PrimeNG CSS link -->
+<h1 class="navbarTitle">{{title}}</h1>
+<main>
```

Comment:
We should generally avoid having ```<main>``` tags in a component since there *should* only be 1 main tag in a document, and we will have many components on screen at once.